### DISCUSSION TITLE
Error: Could not find or load main class JMusicBot.jar Caused by: java.lang.ClassNotFoundException: JMusicBot.jar

### DISCUSSION THREAD

**Discussion by xxtensazenxx** [jagrosh/MusicBot#718] [Questions]
When running the bot as per the wiki I get the following error 

Error: Could not find or load main class JMusicBot.jar
Caused by: java.lang.ClassNotFoundException: JMusicBot.jar

I fixed it by running /usr/bin/java -jar /home/matt/musicbot/JMusicBot.jar -nogui

Just seeing if the method to run via the wiki was incorrect?

**Comment by jagrosh** [comment]
You need to make sure to use the exact name of the jar (including the version), for example
`java -Dnogui=true -jar JMusicBot-0.3.4.jar` (or using the full paths like in your example). Also note that the `-nogui` flag is no longer supported and instead you should use `-Dnogui=true` as in my example.
